admittance_controller/src/admittance_controller.cpp

```diff
@@ -265,6 +283,20 @@ controller_interface::CallbackReturn AdmittanceController::on_configure(
   input_joint_command_subscriber_ =
     get_node()->create_subscription<trajectory_msgs::msg::JointTrajectoryPoint>(
       "~/joint_references", rclcpp::SystemDefaultsQoS(), joint_command_callback);
+
+  input_wrench_command_subscriber_ =
+    get_node()->create_subscription<geometry_msgs::msg::WrenchStamped>(
+      "~/wrench_reference", rclcpp::SystemDefaultsQoS(),
+      [&](const geometry_msgs::msg::WrenchStamped & msg)
+      {
+        if (msg.header.frame_id != admittance_->parameters_.ft_sensor.frame.id && !msg.header.frame_id.empty())
+        {
+          RCLCPP_ERROR_STREAM(
+            get_node()->get_logger(), "Ignoring wrench reference as it is on the wrong frame: " << msg.header.frame_id << ". Expected reference frame: << admittance_->parameters_.ft_sensor.frame_id);
+          return;
```

Comment:
would it make sense to explicitly reset the last wrench to zero in this case?

Reply:
Well that's something which isn't easy to decide. 

I would argue: A wrong input shall not change the state of the system. Therefore we may not change the current offset.

Reply:
Agree with this logic